hclsyntax/expression.go

```diff
@@ -260,6 +260,8 @@ func (e *FunctionCallExpr) Value(ctx *hcl.EvalContext) (cty.Value, hcl.Diagnosti
 		}
 
 		switch {
+		case expandVal.Type().Equals(cty.DynamicPseudoType):
```

Comment:
If passing `null` as the first argument isn't valid, we should have the same null check here as below. Otherwise we have a known value turning into an unknown value.

Reply:
thanks, fixed!